deploy/006-deploy-pool-registry.ts

```diff
@@ -15,20 +15,30 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   );
   const proxyOwnerAddress = await toAddress(preconfiguredAddresses.NormalTimelock || "account:deployer", hre);
 
+  const defaultProxyAdmin = await hre.artifacts.readArtifact(
+    "hardhat-deploy/solc_0.8/openzeppelin/proxy/transparent/ProxyAdmin.sol:ProxyAdmin",
+  );
+
   await deploy("PoolRegistry", {
     from: deployer,
     contract: "PoolRegistry",
     proxy: {
       owner: proxyOwnerAddress,
-      proxyContract: "OpenZeppelinTransparentProxy",
+      proxyContract: "OptimizedTransparentUpgradeableProxy",
       execute: {
         methodName: "initialize",
         args: [accessControlManagerAddress],
       },
+      viaAdminContract: {
+        name: "DefaultProxyAdmin",
+        artifact: defaultProxyAdmin,
+      },
       upgradeIndex: 0,
     },
     autoMine: true,
     log: true,
+    skipIfAlreadyDeployed: true,
+    // maxFeePerGas: "200000000"  // Needed for zksync
```

Comment:
Wouldn't we make this conditional to the network, only setting it for the zksync chains?

Reply:
We need it occasionally. It is due to incorrect fee calculation on zksync sepolia. So, I am removing it as of now.

Reply:
[2837377](https://github.com/VenusProtocol/isolated-pools/pull/423/commits/2837377e9893ca5b18fccf6116314e3fd8f81478)